tests/src/test/java/org/wso2/micro/gateway/tests/common/JMSPublisher.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.micro.gateway.tests.common;
+
+import com.google.gson.JsonObject;
+import io.ballerina.messaging.broker.EmbeddedBroker;
+import org.wso2.micro.gateway.tests.util.ClientHelper;
+
+import javax.jms.Connection;
+import javax.jms.ConnectionFactory;
+import javax.jms.JMSException;
+import javax.jms.Session;
+import javax.jms.Topic;
+import javax.jms.MapMessage;
+import javax.jms.MessageProducer;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.util.Date;
+
+/**
+ * JMS publisher to publish throttled data to throttleData
+ */
+public class JMSPublisher {
+    public static EmbeddedBroker broker = new EmbeddedBroker();
+
+    public void startMessageBroker() throws Exception {
+        broker.start();
+    }
+
+    public void getJson(JsonObject jsonObject) throws JMSException, NamingException {
+        String appKey = jsonObject.getAsJsonObject("event").getAsJsonObject("payloadData").get("appKey").getAsString();
```

Comment:
We can put these common strings to a constant file and use the constants here instead. Ex: "payloadData", "Unauthenticated", "event" etc.

Reply:
Fixed with de596b77c7f9217eca794e47cd29e36dbd24b0d4